pkg/webhook/pod/validating/webhooks.go

```diff
@@ -53,5 +58,12 @@ func (b *podValidateBuilder) Build() admission.Handler {
 	quotaAccessor := quotaevaluate.NewQuotaAccessor(h.Client)
 	h.QuotaEvaluator = quotaevaluate.NewQuotaEvaluator(quotaAccessor, 16, make(chan struct{}))
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.EnablePodEnhancedValidator) {
+		h.PodEnhancedValidator = NewPodEnhancedValidator(b.mgr.GetClient())
```

Comment:
Please note that the controller manager and the informer have not started before the validator's build. So this may cause the validator to get an unsynchronized config.

Reply:
Perhaps it need to get initialized before the validation.